syntax = "proto3";

message InputImage {
    string path = 1;
    int32 part_number = 2;
}

message ImageParts {
    int32 ratio_x = 1;
    int32 ratio_y = 2;
    bool flag = 3;
    repeated string parts = 4;
}

message ImageResult {
    string result_image = 1;
}

service CroppImage {
    rpc cropp(InputImage) returns (ImageParts) {}
}

service ConcatenateImage {
    rpc concatenate(ImageParts) returns (ImageResult) {}
}

